<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyInformation</class>
 <widget class="QDialog" name="MyInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(158, 216, 255);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>显示且用户不可修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>我的信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>显示且用户不可修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前余额：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
